runtime/hsa-runtime/core/runtime/hsa_api_trace.cpp

```diff
@@ -87,7 +87,7 @@ void HsaApiTable::Init() {
   // they can add preprocessor macros on the new functions
 
   constexpr size_t expected_core_api_table_size = 1016;
-  constexpr size_t expected_amd_ext_table_size = 584;
+  constexpr size_t expected_amd_ext_table_size = 592;
```

Comment:
Need to bump HSA_AMD_EXT_API_TABLE_STEP_VERSION

Reply:
Done